package.json

```diff
@@ -20,16 +20,19 @@
   "sideEffects": false,
   "main": "dist/",
   "module": "lib/",
+  "types": "index.d.ts",
```

Comment:
should this be dist/index.d.ts or index.d.ts? I am asking because in postbuild we move index.d.ts into dist. thanks!

Reply:
Sorry, corrected.